docs/Game/Unreal/getting-started/02-sending-transactions.md

```diff
@@ -0,0 +1,346 @@
+---
+title: 02 Sending Transactions & Retrieving Data
+id: unreal-send-retrieve
+---
+
+This page lists the different methods and functions you can make
+
+Send Transaction
+----------------
+----------------
+The **SendTransaction** function interacts with the blockchain and incurs gas fees. Signing in Metamask is required. 
+When a call is made, the back end client sends a 'Ticket' in MetaMask if the call was successful.
+
+### URL 
+
+http://45.77.189.28:5000/send/transaction
+
+### Parameters
+
+`device_id`, 
+`contract_address`, 
+`abi_hash`, 
+`method`, 
+`args`
+
+### Response
+
+A 'ticket' is the response.  
+
+The session saved during **GetClient** is used to open Metamask. Metamask will popup to display gas fees for the transaction. Sign or confirm the transaction for that ticket.
+
+```cpp
+
+void UMirageClient::SendTransaction(FString contract, FString abi_hash, FString method, FString args, FMirageTicket Ticket)
+{
+    http = &FHttpModule::Get();
+
+    TSharedRef<IHttpRequest, ESPMode::ThreadSafe> Request = http->CreateRequest();
+    Request->OnProcessRequestComplete().BindLambda([Ticket, this](FHttpRequestPtr Request, FHttpResponsePtr Response, bool bWasSuccessful)
+        {
+            TSharedPtr<FJsonObject> JsonObject;
+            TSharedRef<TJsonReader<>> Reader = TJsonReaderFactory<>::Create(Response->GetContentAsString());
+            FString data = Response->GetContentAsString();
+
+            if (FJsonSerializer::Deserialize(Reader, JsonObject))
+            {
+                FString ticketId = JsonObject->GetStringField("ticket");
+                data = ticketId;
+            }
+
+            Ticket.ExecuteIfBound(data);
+        });
+
+    AsyncTask(ENamedThreads::AnyBackgroundThreadNormalTask, [this, Request, contract, abi_hash, method, args]()
+        {
+            FString url = baseUrl + "send/transaction";
+
+            Request->SetURL(url);
+            Request->SetVerb("POST");
+            Request->SetHeader(TEXT("User-Agent"), "X-MirageSDK-Agent");
+            Request->SetHeader("Content-Type", TEXT("application/json"));
+            Request->SetContentAsString("{\"device_id\": \"" + deviceId + "\", \"contract_address\": \"" + contract + "\", \"abi_hash\": \"" + abi_hash + "\", \"method\": \"" + method + "\", \"args\": \"" + args + "\"}");
+            Request->ProcessRequest();
+        });
+    #if PLATFORM_ANDROID
+    FPlatformProcess::LaunchURL(session.GetCharArray().GetData(), NULL, NULL);
+    #endif
+    }
+```
+
+Checking Status of Ticket
+------------
+-----------
+
+Check status of tickets by calling **GetTicketResult**.
+
+For example, if you come back to the game, check the status of the ticket with `GetTicketResult` having body parameters as `device_id`, `ticket`.
+
+Returns the status of the result.
+
+### URL
+
+http://45.77.189.28:5000/send/GetTicketResult
+
+### Parameters
+
+`device_id`, `ticket`
+
+### Response
+
+Successful or Unsuccessful.
```

Comment:
If these are literal status codes, `` is missing. Otherwise, why capital in unsuccessful?